pywhat/printer.py

```diff
@@ -87,9 +87,12 @@ def print_json(self, text: dict):
 
     def get_crypto_links(self, text, matched):
         explorers = {
-            "Ethereum Wallet": "https://etherscan.io/address/",
-            "Dogecoin Wallet Address": "https://dogechain.info/address/",
-            "Bitcoin Wallet": "https://www.blockchain.com/btc/address/",
+            "Ethereum (ETH) Wallet Address": "https://etherscan.io/address/",
+            "Dogecoin (DOGE) Wallet Address": "https://dogechain.info/address/",
+            "Bitcoin (BTC) Wallet Address": "https://www.blockchain.com/btc/address/",
+            "Litecoin (LTC) Wallet Address": "https://live.blockcypher.com/ltc/address/",
+            "Bitcoin Cash (BCH) Wallet Address": "https://explorer.bitcoin.com/bch/address/",
+            "Ripple (XRP) Wallet Address": "https://xrpscan.com/account/",

```

Comment:
Nice one! Would be really helpful 🥺  if you picked this up too :-) https://github.com/bee-san/pyWhat/issues/25

Reply:
Please*